src/Common/CosmosClient.ts

```diff
@@ -157,7 +190,6 @@ export function client(): Cosmos.CosmosClient {
 
   const options: Cosmos.CosmosClientOptions = {
     endpoint: endpoint() || "https://cosmos.azure.com", // CosmosClient gets upset if we pass a bad URL. This should never actually get called
-    key: userContext.masterKey,
```

Comment:
Does this affect the emulator?

Reply:
No, I don't think so - we're using EmulatorMasterKey which is hardcoded for the emulator Platform in the Token Provider.